<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>361</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="FinishMainButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Slab,serif</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">float: right;
margin-right: 3%;
margin-top: 5px;
border-radius:5px;
-MOZ-BORDER-RADIUS: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(217,91,72,1);
color: white;
font-family: 'Roboto Slab', serif;
</string>
    </property>
    <property name="text">
     <string>Place an order!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Your order</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>99</width>
      <height>114</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="AddMainButton">
       <property name="styleSheet">
        <string notr="true">float: right;
margin-right: 3%;
margin-top: 5px;
border-radius:5px;
-MOZ-BORDER-RADIUS: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(0, 51, 200, 1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(0, 51, 150, 1);
color: white;
font-family: 'Roboto Slab', serif;
</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DelMainButton">
       <property name="styleSheet">
        <string notr="true">float: right;
margin-right: 3%;
margin-top: 5px;
border-radius:5px;
-MOZ-BORDER-RADIUS: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(0, 51, 200, 1);
text-shadow: #454545 0 0 2px;
border-bottom: 4px solid rgba(0, 51, 150, 1);
color: white;
font-family: 'Roboto Slab', serif;
</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="picLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>280</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Coffee type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>Cappuccino</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Latte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Espresso</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Americano</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Latte Macchiato</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cafe au Lait</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Caffe Mocha</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Topping</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>Cinnamon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Marshmellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extra milk</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_3">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>Small (150 ml)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium (250 ml)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Large (350 ml)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ultimate (500 ml)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>If there is no need to book a table for you, please let us know=).</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string>Take away</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
